### DISCUSSION TITLE
WSS

### DISCUSSION THREAD

**Discussion by VinnieCool** [gorilla/.github#18] [Q&A]
Does it support secure websocket - wss?

**Comment by amustaque97** [comment]
Yes it does support websocket secure, here are a few links to the issue raised in the past on similar lines
https://github.com/gorilla/websocket/issues/844#issuecomment-1666637230
https://github.com/gorilla/websocket/issues/844#issuecomment-1666637230
https://github.com/gorilla/websocket/issues?q=is%3Aissue+secure

I hope this helps, if you have any other queries please feel free to ping back. 
Thanks!
